llvm/lib/CheerpUtils/Registerize.cpp

```diff
@@ -116,13 +116,13 @@ void Registerize::assignRegistersToInstructions(Function& F, cheerp::PointerAnal
 #ifdef VERBOSEDEBUG
 	for(auto it: liveRanges)
 	{
-		if(it.first->getParent()->getParent() != &F)
+		if(it.first.first->getParent()->getParent() != &F)
```

Comment:
can we avoid these `.first.first` by adding some named struct?

Reply:
Changed this to `.first..instruction`